app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -5,6 +5,11 @@ module ForemanAnsible
     # Provider for RemoteExecution that allows to run Ansible playbooks.
     # Read the source of other RemoteExecution providers for more.
     class AnsibleProvider < RemoteExecutionProvider
+      FEATURES_THROUGH_INTERNAL_PROXY = %w(
+        ansible_run_capsule_upgrade
+        ansible_configure_cloud_connector
```

Comment:
Would it make sense to have this as a flag set on feature definition? I don't like hardcoding names here too much. I thought we already have some flags for features.

Reply:
Do we want users to have control over this?